docs/site/angular-auth-oidc-client/docs/migrations/v13-to-v14.md

```diff
@@ -0,0 +1,185 @@
+---
+sidebar_position: 4
+---
+
+# Version 13 to 14
+
+In this version we removed the `APP_INITIALIZER` ([See PR](https://github.com/damienbod/angular-auth-oidc-client/pull/1307)).
+The library now does nothing until you interact with her. No loading of anything and it does not affect your application's bootstrapping process at all.
+You can however explicitly preload your Auth Well-Known Endpoints with a new method called [preloadAuthWellKnownDocument()](/docs/documentation/public-api#preloadauthwellknowndocumentconfigid-string). As a side effect becase the config has to be loaded first, a lot of APIs become reactive and return an Observable now.
+
+All changes are describes below.
+
+## `eagerLoadAuthWellKnownEndpoints` was removed
+
+The Auth Well Known Endpoints are always eager loaded expect loading it explicitly using the [preloadAuthWellKnownDocument()](/docs/documentation/public-api#preloadauthwellknowndocumentconfigid-string) method. No option needed anymore.
+
```

Comment:
# Version 13 to 14

In this version the `APP_INITIALIZER` was removed. ([See PR](https://github.com/damienbod/angular-auth-oidc-client/pull/1307)).
The library will not do anything until the application interact with it. There is no pre-loading of anything and it does not affect your application's bootstrapping process at all.
You can however explicitly preload the secure token server well-known endpoints with a new method called [preloadAuthWellKnownDocument()](/docs/documentation/public-api#preloadauthwellknowndocumentconfigid-string). As a side effect because the config has to be loaded first, a lot of APIs become reactive and return an Observable now.

All changes are described below.

## `eagerLoadAuthWellKnownEndpoints` was removed

The secure token server well known endpoints are always eager loaded expect loading it explicitly using the [preloadAuthWellKnownDocument()](/docs/documentation/public-api#preloadauthwellknowndocumentconfigid-string) method. No option needed anymore.